packages/theme/src/index.ts

```diff
@@ -16,21 +16,43 @@
 
 import { elyraIcon } from '@elyra/ui-components';
 import {
+  ILabShell,
   JupyterFrontEnd,
   JupyterFrontEndPlugin
 } from '@jupyterlab/application';
+import { ICommandPalette, MainAreaWidget } from '@jupyterlab/apputils';
+import { ILauncher, LauncherModel } from '@jupyterlab/launcher';
+import { launcherIcon } from '@jupyterlab/ui-components';
 import { IMainMenu } from '@jupyterlab/mainmenu';
 
+import { toArray } from '@lumino/algorithm';
+import { Widget } from '@lumino/widgets';
+
+import { CustomLauncher } from './launcher';
 import '../style/index.css';
 
+/**
+ * The command IDs used by the launcher plugin.
+ */
+const CommandIDs = {
+  create: 'launcher:create-new'
```

Comment:
```suggestion
  create: 'launcher:create'
```

Reply:
This needs to match the command that's normally used for the launcher (and the launcher extension has to be disabled) for this to work.